python/src/GaussianProcessFitterResult_doc.i.in

```diff
@@ -0,0 +1,194 @@
+%feature("docstring") OT::GaussianProcessFitterResult
+"Gaussian process fitter result.
+
```

Comment:
You might want to replace p with \outputDim throughout this file.

Reply:
Bof
I used the same notations than the actual implementation (see [this](https://openturns.github.io/openturns/latest/user_manual/response_surface/_generated/openturns.KrigingAlgorithm.html#openturns.KrigingAlgorithm) for example)